libraries/DNSServer/examples/CaptivePortal/CaptivePortal.ino

```diff
@@ -3,7 +3,7 @@
 #include <ESP8266WebServer.h>
 
 const byte DNS_PORT = 53;
-IPAddress apIP(192, 168, 1, 1);
+IPAddress apIP(172, 217, 28, 1);
```

Comment:
This PR doesn't make a lit of sense.

192.168.x.x is a private class B subnet, 172.x.x.x is a private class C subnet.  Both are equally valid AP addresses, and should have no effect upon DHCP clients who obey the relevant standards, which I certainly hope all OSes, mobile or otherwise, do.